src/directives/toaster.directive.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  id: 'toaster',
+  bind () {
+    /* eslint no-undef: 0 */
```

Comment:
Should use `/* eslint-disable-next-line`

`/* eslint ` disable rule for all project